pkg/controllers/management/kontainerdrivermetadata/metadata_handler.go

```diff
@@ -120,6 +121,8 @@ func (m *MetadataController) sync(key string, setting *v3.Setting) (runtime.Obje
 		m.Settings.Controller().EnqueueAfter(setting.Namespace, setting.Name, time.Minute*time.Duration(interval))
 	}
 
+	// refresh to sync k3s/rke2 releases
+	channelserver.Refresh()
```

Comment:
make channel is called (https://github.com/rancher/rancher/blob/release/v2.6/pkg/channelserver/channelserver.go#L124) as a part of https://github.com/rancher/rancher/blob/release%2Fv2.6/pkg/multiclustermanager/routes.go#L69 before controllers are started so this looks safe. 👍🏼